plugins/topology/src/components/Topology/TopologySideBar/PodLogs/PodLogs.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { DismissableBanner, LogViewer } from '@backstage/core-components';
+
+import { Paper } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+
+import { usePodLogs } from '../../../../hooks/usePodLogs';
+import { ContainerScope } from './types';
+
+type PodLogsProps = {
+  podScope: ContainerScope;
+};
+
+export const PodLogs = ({ podScope }: PodLogsProps) => {
+  const [loadingData, setLoadingData] = React.useState(true);
+  const { value, error, loading } = usePodLogs({
+    podScope: podScope,
+  });
+
+  React.useEffect(() => {
+    if (!loading && loadingData) {
+      setLoadingData(false);
+    }
+  }, [loading, loadingData]);
```

Comment:
Why can we not use `loading` itself instead of updating another variable based on its value and then using it?

Reply:
you are right it's clean to move this usePodsLogs and the caller can rely on loading but the local state is needed as am making calls in intervals (used `useAsyncRetry`) and for each call loading turns to true and then false